### ISSUE TITLE
Help?

### ISSUE THREAD

**Issue by ImTheDeveloper** [cristijora/vue-now-ui#1]
Great work on this so far. I have actually just integrated now-ui with bootstrap-vue, it works but not great and then I stumbled upon this repo. If you need any help to get this finished let me know.

**Comment by cristijora** [comment]
@ImTheDeveloper Any help would be greatly appreciated! I parked development on this kit for later on because I have to finish some PRO dashboard first. 

You can see some issues in the roadmap https://github.com/cristijora/vue-now-ui/projects/1 
It should mainly match http://demos.creative-tim.com/now-ui-kit/documentation/tutorial-components.html

Some easy components to start with could be: 
### Modal, Tooltip, Tabs (you can find inspiration https://github.com/cristijora/vue-tabs)

For the api of these components, I try to have a combination of element-ui API for components (props, events etc) and vuetify so we can have a very clean api for components which is intuitive.

Development tips:
If you add a new component, make sure to add it under `components.json` https://github.com/cristijora/vue-now-ui/blob/master/components.json and make sure it has an `index.js` and a separate folder. This would allow to build separate files and have high modularity so people can import only the stuff they need.

To add documentation for a component make sure to add the navigation for it https://github.com/cristijora/vue-now-ui/blob/master/documentation/nav.config.json

And then add the corresponding makrdown file. See for example button 
https://github.com/cristijora/vue-now-ui/edit/master/documentation/docs/en-US/button.md

A special `:::demo  ::::` block will  basically parse the vue code inside it and generate a fiddle which for the moment doesn't work because the kit is not published to npm but don't worry about it, it will be fixed.
If you have state for your components, make sure to add it before in a script tag
https://github.com/cristijora/vue-now-ui/edit/master/documentation/docs/en-US/checkbox.md 
because it won't work in the demo blocks otherwise.

That's pretty much it. If you have time, unit tests would be very handy. Also, you can write if you get blocked by something or don't know how to handle something. Pretty much, only modifying the files mentioned above should be enough to add more components or to fix something in the existing ones. Build setup and all the other stuff is already done so there is not much to do there. Thanks!

**Comment by ImTheDeveloper** [comment]
Not a problem - thanks for the pointers and extensive info. As I get time I will throw some PR's forward. I know on the discord chat there have been loads of people asking for help on now-ui integration so I think this could be really useful to them.

I'm currently using Nuxt.js so I'll take a look at some install/integration documentation for that also as I see you are focusing on just the webpack side so far. There's nuxt community plugins which I know bootstrap-vue is installed through heavily to integrate with nuxt, hopefully I can produce something like that for now-ui too.

**Comment by cristijora** [comment]
I tried to focus on the basics so far. Nuxt and SSR can be done once we have all the components.
